mhkit/wave/performance.py

```diff
@@ -318,9 +319,9 @@ def mean_annual_energy_production_matrix(LM, JM, frequency):
         Mean annual energy production
 
     """
-    LM = convert_to_dataarray(LM)
-    JM = convert_to_dataarray(JM)
-    frequency = convert_to_dataarray(frequency)
+    LM = convert_to_dataset(LM).to_array()
+    JM = convert_to_dataset(JM).to_array()
+    frequency = convert_to_dataset(frequency).to_array()
```

Comment:
This change is causing the tests to fail. I'm looking into it but if you are around @akeeste I know you are more familiar with these changes.

Reply:
@akeeste can you tell me the reasoning behind this change? Is it simply a performance thing to go Dataset to array instead just to array?

Reply:
The easiest solution is to skip the conversion if it is already a DataArray. Otherwise we are going to need to pass the name around to fix this.

Do you have a preference?

Reply:
@ssolson I fixed these few lines. To be honest, I can't recall at the moment why I made this change, but I reverted it and the power_performance_workflow tests are passing now.